main/commit-msg

```diff
@@ -1,12 +1,16 @@
-#!/usr/bin/env python3
-'''
-A hook to check commit massage.
+#!/usr/bin/env bash
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 
-This is currently used to decide if large file should be commited.
```

Comment:
These changes look sensible, but I feel they should be removed from this PR as it makes it impossible to merge it to main as it replaces the python calls but without replacing them with a working C# version.  I feel it would be better to be able to merge the C# to main independently of these changes to make them the default.